packages/design-tokens/src/tokens/functional/components/bento/base.json

```
@@ -0,0 +1,43 @@
+{
+  "brand": {
+    "Bento": {
+      "gap": {
+        "large": {
+          "value": "var(--base-size-32)"
+        },
+        "medium": {
+          "value": "var(--base-size-24)"
```

Medium should be 32px + not sure we need a large treatment here as they have the same values

```suggestion
          "value": "var(--base-size-32)"
```